docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,171 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 15 December 2023
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
+For this reason, they are not subject to the same block delays and data space constraints as the main Tezos network.
 
-For examples of Smart Rollups, see this repository: https://gitlab.com/tezos/kernel-gallery.
+The transactions and logic that Smart Rollups run is called _layer 2_ to differentiate it from the main network, which is called _layer 1_.
+
+The compiled code of a Smart Rollup must be provided publicly, so that anyone can run a Smart Rollup node to execute that code and verify that other nodes are running it correctly, just like anyone can run nodes, bakers, and accusers on layer 1.
+This code, called the _kernel_, must behave in a deterministic manner and according to a given semantics, which guarantees that results are reproducible by any rollup node with the same kernel.
+The semantics is precisely defined by a reference virtual machine called a proof-generating virtual machine (PVM), able to generate a proof that executing a program in a given context results in a given state.
+This guarantees that if a divergence in results is found, it can be tracked down to a single elementary step that was not executed correctly by some node.
+In this way, multiple nodes can run the same code to create a single state for the rollup, and other nodes can verify the state of the rollup.
+
+For a tutorial on Smart Rollups, see [Deploy a Smart Rollup](../tutorials/smart-rollup).
+
+For reference on Smart Rollups, see [Smart Optimistic Rollups](https://tezos.gitlab.io/active/smart_rollups.html) in the Octez documentation.
+
+This diagram shows a high-level view of how Smart Rollups interact with layer 1:
+
+<LucidDiagram width="640px" height="480px" src="https://lucid.app/documents/embedded/0502acd1-ac3a-45a5-b3a6-8b0d77d6e951" id="BE6RELfrN5xE" />
+
+## Uses for Smart Rollups
+
+Smart Rollups allow you to run large amounts of processing that would be too slow or expensive to run on layer 1.
+Thus they allow you to avoid some transaction fees and storage fees.
+For example, Smart Rollups are not constrained to the layer 1 block baking rate and therefore can run far more transactions per second than layer 1.
+Smart Rollups can also manipulate large amounts of data without needing to store it on layer 1.
```

Comment:
"manipulating date" and "getting data from outside the blockchain" are different use cases, so I don't think they belong together. I think "manipulating large amounts of data" fits better in the paragraph about "run large amounts of processing that would be too slow or expensive to run on layer 1" because you can manipulate large amounts of data on layer 1, it would just be cost prohibitive.

How about this:

> Smart Rollups allow you to run large amounts of processing and manipulate large amounts of data that would be too slow or expensive to run on layer 1. Thus they allow you to avoid some transaction fees and storage fees. Also, Smart Rollups are not constrained to the layer 1 block baking rate and therefore can run far more transactions per second than layer 1.

Reply:
> "manipulating [large] data" and "getting data from outside the blockchain" are different use casesI

Maybe, although the former is a consequence of the latter. OK, let's group this like you do. But then let's make 4 use cases:
1. heavy computations and/or large data
2. high transaction throughput
3. data from outside
4. implementing other execution environments

Then, all what you have to do is add a blank line in this paragraph before "Also" (and maybe remove the "Also").

Reply:
Made this into a bulleted list and added the "avoid fees" use case:

- Smart Rollups allow you to run large amounts of processing and manipulate large amounts of data that would be too slow or expensive to run on layer 1.

- Smart Rollups are not constrained to the layer 1 block baking rate and therefore can run far more transactions per second than layer 1.

- Smart Rollups allow you to avoid some transaction fees and storage fees.

- Smart Rollups can retrieve data from outside the blockchain in specific ways that smart contracts can't do.

- Smart Rollups can implement different execution environments, such as execution environments that are compatible with other blockchains. For example, Smart Rollups enable [Etherlink](https://www.etherlink.com/), which makes it possible to run EVM applications (originally written for Ethereum) on Tezos.